manifests/prow/ingresses/prow-hook.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    external-dns.alpha.kubernetes.io/hostname: prow-hook.mgmt.3sca.
+    external-dns.alpha.kubernetes.io/target: prow-hook.apps.pro-base-ocp4.mgmt.3sca.net
```

Comment:
What's the difference between external-dns hostname/target on these 2 lines?

Reply:
This service is a core service, not tied to a particular cluster and public. So this will tell external-dns to create a CNAME in the `mgmt.3sca.net` public zone: `prow-hook.mgmt.3sca.net` pointing  to the cluster private zone domain at `prow-hook.apps.pro-base-ocp4.mgmt.3sca.net`. The `prow-hook.apps.pro-base-ocp4.mgmt.3sca.net` is served by the load balancer behind wildcard `*.apps..pro-base-ocp4.mgmt.3sca.net` deployed as the OpenShift default router. 

https://github.com/kubernetes-sigs/external-dns/blob/master/docs/faq.md#are-other-ingress-controllers-supported

Reply:
understood!